datafusion/proto/src/physical_plan/from_proto.rs

```diff
@@ -110,7 +114,9 @@ pub fn parse_physical_sort_exprs(
         .iter()
         .map(|sort_expr| {
             if let Some(expr) = &sort_expr.expr {
-                let expr = parse_physical_expr(expr.as_ref(), registry, input_schema)?;
+                let codec = DefaultPhysicalExtensionCodec {};
+                let expr =
+                    parse_physical_expr(expr.as_ref(), registry, input_schema, &codec)?;
```

Comment:
I create `DefaultPhysicalExtensionCodec` for cases that will not need `ExtensionCodec`